README.md

```diff
@@ -7,3 +7,29 @@ for an overview of what this tool will do.
 
 It is currently under active development and is not ready for production (or
 even development) use, but stay tuned for updates!
+
+### Migrating from Halyard
+
+A Halyard-generated halconfig requires several manual adjustments before it can
+be used as input to Kleat.
+
+#### Required Changes
+
+- A Halyard-generated halconfig contains two top-level fields:
+`deploymentConfigurations` (a list of `HalConfig`s) and `currentDeployment`,
+(the name of the `deploymentConfiguration` to manage). Kleat accepts a file with
+a single `HalConfig` as input, so any halconfigs with multiple
+`deploymentConfigurations` need to be separated into different files.
+- Halyard specified
+`HalConfig.Providers.AzureProvider.AzureAccount.useSshPublicKey` as a `string`
+with `'true'` and `'false'` as acceptable values. Kleat requires this field to
+be specified as a `bool`.
+
+#### Optional Changes
+
+- Halyard provided a default value for every possible member of the halconfig.
+Kleat neither requires nor supplies any default values, which means it is
+acceptable to remove all values for which the behavior of an unspecified
```

Comment:
I wonder if something like "for which the microservice's deafult value is acceptable" would be more clear?  Or maybe people  shouldn't need to know exactly what fields affect each microservice?  I just found "behavior of an unspecified field" a bit confusing on first read.

Reply:
> I wonder if something like "for which the microservice's deafult value is acceptable" would be more clear?

Thanks, that's a lot clearer, updating! And I'm not sure how much people should have to know exactly what fields affect each microservice: in an ideal future world, perhaps the `Halconfig` proto lives in Kleat, but the `Clouddriver` proto and all its Clouddriver-only messages are exported from Clouddriver and live close to the code that will default them.